File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/generationConfig/capitalizedGetterSetterNames/CapitalizedGetterNames_ViewModel.cpp

#include "CapitalizedGetterNames_ViewModel.h"
#include <vector>
#include <memory>
#include "CapitalizedGetterNames_ViewModelMyElementsRow.h"
#include <string>

bool CapitalizedGetterNames_ViewModel::GetIsMyFlagCheckBoxChecked( ) 
{
  return this->isMyFlagCheckBoxChecked;
}

bool CapitalizedGetterNames_ViewModel::GetMyVisibility( ) 
{
  return this->myVisibility;
}

bool CapitalizedGetterNames_ViewModel::GetIsMyFlagCheckBoxEnabled( ) 
{
  return this->isMyFlagCheckBoxEnabled;
}

std::string CapitalizedGetterNames_ViewModel::GetMyFlagCheckBoxText( ) 
{
  return this->MyFlagCheckBoxText;
}

std::vector<std::shared_ptr<CapitalizedGetterNames_ViewModelMyElementsRow>> CapitalizedGetterNames_ViewModel::GetMyRows( ) 
{
  return this->myRows;
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/generationConfig/capitalizedGetterSetterNames/CapitalizedGetterNames_ViewModel.cs
/// <filename>
///     CapitalizedGetterNames_ViewModel.cs
/// </filename>
using System.Collections.Generic;

public class CapitalizedGetterNames_ViewModel
{
    private bool isMyFlagCheckBoxChecked;

    public virtual bool GetIsMyFlagCheckBoxChecked()
    {
        return this.isMyFlagCheckBoxChecked;
    }

    private bool myVisibility;

    public virtual bool GetMyVisibility()
    {
        return this.myVisibility;
    }

    private bool isMyFlagCheckBoxEnabled;

    public virtual bool GetIsMyFlagCheckBoxEnabled()
    {
        return this.isMyFlagCheckBoxEnabled;
    }

    private string MyFlagCheckBoxText;

    public virtual string GetMyFlagCheckBoxText()
    {
        return this.MyFlagCheckBoxText;
    }

    private System.Collections.Generic.List<CapitalizedGetterNames_ViewModelMyElementsRow> myRows = new System.Collections.Generic.List<CapitalizedGetterNames_ViewModelMyElementsRow>();

    public virtual System.Collections.Generic.List<CapitalizedGetterNames_ViewModelMyElementsRow> GetMyRows()
    {
        return this.myRows;
    }
}
